listenbrainz/spark/handlers.py

```diff
@@ -4,7 +4,25 @@
 import listenbrainz.db.user as db_user
 import listenbrainz.db.stats as db_stats
 
-from flask import current_app
+from flask import current_app, render_template
+from brainzutils.mail import send_mail
+from datetime import datetime, timezone, timedelta
+
+TIME_TO_CONSIDER_STATS_AS_OLD = 12 # hours
+
+def new_user_stats():
```

Comment:
this function name doesn't exactly make it clear what is going on. Stats for a new user? Or new stats about users?